packages/e2e/features/ui/components/storage/storage-browser/navigate-locations.feature

```diff
@@ -8,36 +8,35 @@ Feature: Storage Browser navigate breadcrumbs
     When I type my "email" with status "CONFIRMED"
     Then I type my password
     Then I click the "Sign in" button
-    When I click the button containing "public"
+    When I click the first button containing "public"
     Then I see the "Home" button
     When I click the "Home" button
     Then I see "HOME"
-    Then I see the button containing "public"
+    Then I see the first button containing "public"
 
   @react
   Scenario: Navigate back up to prefix
     When I type my "email" with status "CONFIRMED"
     Then I type my password
     Then I click the "Sign in" button
-    When I click the button containing "public"
-    Then I see the "Blueberry/" button
-    When I click the "Blueberry/" button
-    Then I see "Blackberry/"
-    When I click the "Blackberry/" button
-    Then I see "Blueberry"
-    When I click the "Blueberry" button
-    Then I see "public/Blueberry/"
+    When I click the first button containing "public"
```

Comment:
Changing integ test: Since some folder for test were deleted. so modify to test only for this